web/docs/project/env-vars.md

```diff
@@ -92,40 +93,44 @@ These files should not be committed to version control, and they are already ign
 ### 2. Using Shell
 If you set environment variables in the shell where you run your Wasp commands (e.g., `wasp start`), Wasp will recognize them.
 
-You can set environment variables in the `.profile` or a similar file, or by defining them at the start of a command:
-
-```shell
-SOME_VAR_NAME=SOMEVALUE wasp start
-```
+You can set environment variables in the `.profile` or a similar file, which will set them permanently, or you can set them temporarily by defining them at the start of a command (`SOME_VAR_NAME=SOMEVALUE wasp start`).
 
- This is not specific to Wasp and is simply how environment variables can be set in the shell.
+This is not specific to Wasp and is simply how environment variables can be set in the shell.
 
-Defining environment variables in this way can be cumbersome even for a single project and even more challenging to manage if you have multiple Wasp projects. Therefore, we do not recommend this as a default method for providing environment variables to Wasp projects. However, it can be useful for occasionally **overriding** specific environment variables because environment variables set this way **take precedence over those defined in `.env` files**.
+Defining environment variables in this way can be cumbersome even for a single project and even more challenging to manage if you have multiple Wasp projects. Therefore, we do not recommend this as a default method for providing environment variables to Wasp projects during development, you should use .env files instead. However, it can be useful for occasionally **overriding** specific environment variables because environment variables set this way **take precedence over those defined in `.env` files**.
```

Comment:
```suggestion
Defining environment variables inline can be cumbersome even for a single project and even more challenging to manage if you have multiple Wasp projects.
Therefore, we do not recommend this as a default method for providing environment variables to Wasp projects during development. Instead, use the `.env` files instead.
Inline definitions are useful for occasionally **overriding** specific environment variables because environment variables set this way **take precedence over those defined in `.env` files** (list example perhaps?)
```

Reply:
While "in this way" refers to the context above, which is not ideal, what you did with "inline" is likely worse, as it is a new concept we haven't mentioned so far and you haven't explained it here, so it is not clear to what you are referring.

But similar like above, I wouldn't go into refactoring the whole paragraphs here when we just added one small piece of the sentence, let's leave that for another PR.